features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -225,6 +215,17 @@
     <bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial.linuxsysfs/${project.version}</bundle>
   </feature>
 
+  <feature name="openhab-core-ephemeris" version="${project.version}">
+    <requirement>openhab.tp;filter:="(feature=jollyday)"</requirement>
+    <feature dependency="true">openhab.tp-jollyday</feature>
+    <feature>openhab-core-base</feature>
+    <bundle>mvn:org.openhab.core.bundles/org.openhab.core.ephemeris/${project.version}</bundle>
+    <config name="org.openhab.ephemeris">
```

Comment:
Has `org.openhab.ephemeris` been choosen by intention (the `core` is missing between openhab and ephemeris)?

Reply:
Yes, this is the standard scheme for any configuration settings - as a result, you can define the parameters in a file called `ephemeris.cfg` and don't need to deal with any namespaces.